### PR TITLE
Fix publisher handle variable name

### PR DESCRIPTION
**Pull request by esteve**
This fixes `main` not building due to #194

### CODE REVIEW

**Review by jhdcs — APPROVED** [review]

**Review by nnmm — APPROVED** [review]

**Comment by jhdcs** [comment]
The tests appear to be failing from an internal server error where the repositories are stored. Should we wait for that to get 
resolved before merging?

**Comment by esteve** [comment]
@jhdcs @nnmm thanks for the prompt reviews, I'll wait until CI passes, but it looks like there's an issue with docker.io being down.

**Pull request merged**
